Feature : MultiSong

vijayrawatsan opened the pull request:
- Earlier the script detected the first song of the search. Now it asks users to download from a list of alternatives found on the first page of search.
- Earlier script was crashing when no results were found. Try to search njaania. Now its fixed.

Screenshot below:

<img width="910" alt="instantmusic" src="https://cloud.githubusercontent.com/assets/985228/10399326/a74573ca-6ecf-11e5-8d34-df0bfd17a27a.png">

yask123 commented:
This feature has already been added f6ca5bfdb8f4244e8d7a44633bbb5e262eef866f .
I haven't updated the python module as yet, thats why it didn't reflect in `/bin/instantmusic`
If there is something else your script fixes, let me know.

vijayrawatsan commented:
Okay. This script also fixed a crash when no search results are found. I hope that is also fixed in https://github.com/yask123/Instant-Music-Downloader/commit/f6ca5bfdb8f4244e8d7a44633bbb5e262eef866f
